code/mob/living/silicon.dm

```diff
@@ -28,6 +28,9 @@ ADMIN_INTERACT_PROCS(/mob/living/silicon, proc/pick_law_rack)
 
 	var/static/regex/monospace_say_regex = new(@"`([^`]+)`", "g")
 
+	var/hat_offset_y = 0
+	var/hat_offset_x = 0
```

Comment:
Not sure what these are doing here, if you want to do specific offsets per type then pass the offsets in the component init.